android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -282,27 +335,89 @@ public void onError(String info) {
      * @param parameters         streaming parameters to be used when streaming. If null is sent in, the default/optimized options will be used.
      *                           If you are unsure about what parameters to be used it is best to just send null and let the system determine what
      *                           works best for the currently connected module.
-     * @param encrypted          a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
-     */
-    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted) {
-        this.context = new WeakReference<>(context);
-        this.remoteDisplayClass = remoteDisplayClass;
-        int majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
-        boolean isCapabilitySupported = internalInterface.getSystemCapabilityManager() != null && internalInterface.getSystemCapabilityManager().isCapabilitySupported(SystemCapabilityType.VIDEO_STREAMING);
-        if (majorProtocolVersion >= 5 && !isCapabilitySupported) {
-            DebugTool.logError(TAG, "Video streaming not supported on this module");
-            stateMachine.transitionToState(StreamingStateMachine.ERROR);
-            return;
-        }
+	 * @param encrypted a flag of if the stream should be encrypted. Only set if you have a supplied encryption library that the module can understand.
+	 * @param portraitRange constraints for vehicle display : aspect ratio, min/max resolutions, max diagonal size.
```

Comment:
@kostyaBoss I would swap lines 339 and 340 since landscapeRange is passed into the method before portraitRange.